### DISCUSSION TITLE
window.ethereum injected wallet options not showing up

### DISCUSSION THREAD

**Discussion by shazow** [WalletConnect/walletconnect-monorepo#3159] [Developer Support] [answered]
Hi there,

When I hit the `Connect Wallet` button on https://web3modal.com/, it shows MetaMask (INSTALLED) as desired. But when I run an example locally, I'm not seeing the installed MetaMask option. Am I doing something wrong?

I found some old links to docs at https://docs.walletconnect.com/2.0/web3modal/options but this link is broken now. Is there other docs somewhere?

Using `"@walletconnect/ethereum-provider@^2.9.0"` and `"@walletconnect/modal@^2.6.0"`

Relevant code:

```javascript
import { EthereumProvider } from '@walletconnect/ethereum-provider';

...

      const provider = await EthereumProvider.init({
        projectId: projectId,
        showQrModal: true,
        infuraId: infuraId,
        qrModalOptions: { themeMode: "light" },
        chains: [1],
        optionalChains: [11155111], // Sepolia
        metadata: metadata,
      });

...
      provider.connect();
```

Feels like I'm doing something wrong, what am I missing? Thanks!

**Comment by glitch-txs** [comment, marked as answer]
Hi those are different packages.

Ethereum-provider != Web3modal

Ethereum-provider doesn't work with injected providers

**Reply by shazow** [reply]
Ah my bad, I thought the qrModal was powered by web3modal
